smoke/IotEdgeQuickstart/details/IotedgedLinux.cs

```diff
@@ -67,22 +67,112 @@ static string GetIpAddress()
         }
     }
 
+    interface ILinuxPackageInstall
+    {
+        public Task Install();
+        public Task FindPackage(string packageName);
+        public Task RemovePackage(string packageName);
+    }
+
+    class LinuxPackageInstallDep : ILinuxPackageInstall
+    {
+        readonly string archivePath;
+        public LinuxPackageInstallDep(string archivePath)
+        {
+            this.archivePath = archivePath;
+        }
+
+        public Task Install()
+        {
+            string[] packages = Directory.GetFiles(this.archivePath, "*.deb");
+
+            foreach (string package in packages)
+            {
+                Console.WriteLine($"Will install {package}");
+            }
+
+            string packageArguments = string.Join(" ", packages);
+
+            return Process.RunAsync(
+                "apt-get",
+                $"install -y {packageArguments}",
+                300); // 5 min timeout because install can be slow on raspberry pi
```

Comment:
1. Please consolidate the string `apt-get` or `apt` into a string const var and use that for the different functions.
2. Let's go ahead and create a generic timeout const for apt commands to be used across `apt-get install`, `apt-get pruge`, etc.
3. Please do the same for RPM
4. Do we want to use RPM or dnf or yum?  What's the doc from Mariner team suggest?
5. Thank you for refactoring :)

Reply:
mariner normally uses tdnf for all package interaction, but rpm has all the functionality we need  here since we are not doing any type of dependency management. I will use whichever you prefer.

Reply:
note, Mariner has rpm, yun, dnf and tdnf